packages/discussions/graphql/resolvers/_mutations/suspendUser.js

```diff
@@ -0,0 +1,25 @@
+const { Roles } = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, args, context) => {
+  const { pgdb, loaders, user: me } = context
+
+  Roles.ensureUserIsInRoles(me, ['admin', 'supporter'])
+
+  const tx = await pgdb.transactionBegin()
+  try {
+    await pgdb.public.discussionSuspensions.insert({
+      userId: args.id,
+      beginAt: new Date(),
+      ...(args.until && { endAt: args.until }),
```

Comment:
Am I missing the week's default interval? I can't see it

Reply:
That is [declared in database migration](https://github.com/orbiting/backends/pull/550/files#diff-c623ab3eaade6dfcf122c168b16926eca18f4fe039d29e4021e3e4640b8796ddR5). Just to have no useless records.

But a coded default, too, is more understandable, now that I think of it. 👍

Reply:
aaaaah, ok – you've got to have eagle eyes to spot that!! 🦅